test/configs/simple.toml

```diff
@@ -0,0 +1,21 @@
+[relaychain]
```

Comment:
I presume this would replace [this file here](https://github.com/Moonsong-Labs/storage-hub/blob/2fa2da608da53b4f0e6a0b3cd67a7966dc4339ee/zombienet/config.toml#L1), right? If so, let's delete the `zombienet` folder at the root of the project to prevent ambiguity.